website/source/docs/configuration/interpolation.html.md

```diff
@@ -168,6 +168,11 @@ The supported built-in functions are:
       `n` is the index or name of the subcapture. If using a regular expression,
       the syntax conforms to the [re2 regular expression syntax](https://code.google.com/p/re2/wiki/Syntax).
 
+  * `signum(int)` - Returns -1 for negative numbers, 0 for 0 and 1 for positive numbers.
+      This function is useful when you need to set a value for the first resource and
+      a different value for the rest of the resources.
+      Example: `element(split(",", var.r53_failover_policy), signum(count.index))`
```

Comment:
Would you mind also mentioning the example from the issue? I think that would help users to better understand possible use-cases of this function.

i.e. I'd change this

> Example: `element(split(",", var.r53_failover_policy), signum(count.index))`

to

> Example: `element(split(",", var.r53_failover_policy), signum(count.index))` where the zeroth index points to `PRIMARY` and 1st to `FAILOVER`